File: Fibonacci Sequence.py
def fibonacci(n):
    if n <= 0:
        print("The number must be positive.")  # Check if the number is positive
        return
    fib_sequence = [0, 1]  # Start of the Fibonacci sequence
    while len(fib_sequence) < n:
        next_number = fib_sequence[-1] + fib_sequence[-2]  # Calculate the next number in the sequence
        fib_sequence.append(next_number)  # Add the next number to the sequence
    return fib_sequence[:n]  # Slice the sequence to get the first n numbers

# Test the function
num = int(input("Please enter a positive integer: "))
sequence = fibonacci(num)
if sequence:
    print("Fibonacci sequence:")
    print(sequence)

File: Fibonacci Sequence.js
function fibonacci(n) {
    if (n <= 0) {
      console.log("The number must be positive.");  // Check if the number is positive
      return;
    }
    var fibSequence = [0, 1];  // Start of the Fibonacci sequence
    while (fibSequence.length < n) {
      var nextNumber = fibSequence[fibSequence.length - 1] + fibSequence[fibSequence.length - 2];  // Calculate the next number in the sequence
      fibSequence.push(nextNumber);  // Add the next number to the sequence
    }
    return fibSequence.slice(0, n);  // Slice the sequence to get the first n numbers
  }
  
  // Test the function
  var num = parseInt(prompt("Please enter a positive integer: "));
  var sequence = fibonacci(num);
  if (sequence) {
    console.log("Fibonacci sequence:");
    console.log(sequence);
  }